backend/functions/orders/orders.function.ts

```diff
@@ -1,65 +1,49 @@
-import { AzureFunction, Context, HttpRequest } from '@azure/functions';
 import * as dotenv from 'dotenv';
-
-import { httpError, httpSuccess } from '../lib/adapters/utils/http-response';
-import ContextCreator from '../lib/adapters/utils/application-context-creator';
+import { app, HttpRequest, HttpResponseInit, InvocationContext } from '@azure/functions';
+import ContextCreator from '../azure/application-context-creator';
 import { initializeApplicationInsights } from '../azure/app-insights';
-import {
-  OrdersController,
-  GetOrdersResponse,
-  PatchOrderResponse,
-} from '../lib/controllers/orders/orders.controller';
-import { BadRequestError } from '../lib/common-errors/bad-request';
-import { TransferOrderAction } from '../../../common/src/cams/orders';
-import { ApplicationContext } from '../lib/adapters/types/basic';
+import { OrdersController } from '../lib/controllers/orders/orders.controller';
+import { Order, TransferOrderAction } from '../../../common/src/cams/orders';
+import { toAzureError, toAzureSuccess } from '../azure/functions';
 
 const MODULE_NAME = 'ORDERS_FUNCTION';
 
 dotenv.config();
 
 initializeApplicationInsights();
 
-const httpTrigger: AzureFunction = async function (
-  functionContext: Context,
+export default async function handler(
   request: HttpRequest,
-): Promise<void> {
-  const context = await ContextCreator.applicationContextCreator(functionContext, request);
-  let response;
+  invocationContext: InvocationContext,
+): Promise<HttpResponseInit> {
+  const logger = ContextCreator.getLogger(invocationContext);
+  let response: HttpResponseInit;
   try {
+    const context = await ContextCreator.applicationContextCreator(invocationContext, request);
     context.session = await ContextCreator.getApplicationContextSession(context);
-
-    if (request.method === 'GET') {
-      response = await getOrders(context);
-    } else if (request.method === 'PATCH') {
-      response = await updateOrder(context);
+    const orderController = new OrdersController(context);
+    if (context.request.method === 'GET') {
+      const orderGet = await orderController.getOrders(context);
+      response = toAzureSuccess<Order[]>(orderGet);
+    } else if (context.request.method === 'PATCH') {
+      //TODO: Json Mapping with these requestBody objects
+      const id = context.request.params['id'];
+      const camsResponse = await orderController.updateOrder(
+        context,
+        id,
+        context.request.body as TransferOrderAction,
+      );
```

Comment:
Two comments regarding this TODO. First, we should not be splitting the request parameters out anymore before calling the controller. Second, if the mapping idea is to be able to turn `context.request.params['id']` into `context.request.params.id`, we should already be able to do that (for example see `orders.controller.ts`).

Reply:
um... is there any difference between context.request.params['id'] and context.request.params.id?  Isn't that literally the same thing?  Or is params an actual array?